.vscode/settings.json

```diff
@@ -43,5 +43,8 @@
 	"workbench.editorAssociations": {
 		"*.dmi": "dmiEditor.dmiEditor"
 	},
-	"Lua.diagnostics.enable": false
+	"Lua.diagnostics.enable": false,
+	"githubPullRequests.ignoredPullRequestBranches": [
```

Comment:
What is this change?

Reply:
Again I'm sorry still new to github.  Didn't want to include this I have no idea how to uncommit this. U can remove this change